packages/cozy-client/src/index.js

```diff
@@ -37,7 +37,7 @@ export { getQueryFromState } from './store'
 export { default as Registry } from './registry'
```

Comment:
Inside this commit message, you should write how to handle the BC.

Something like: 
```
If you were importing manifest from 'cozy-client/dist/manifest` you have to either import it from the root .... or from the models.